README.md

```diff
@@ -0,0 +1,23 @@
+# org.polymc.PolyMC
```

Comment:
Just a note: users are unlikely to visit this file. Usually you want to include as much as such info in the app itself or metainfo file.

Reply:
added some more info to meta